src/content/translations/de/developers/docs/consensus-mechanisms/pos/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Proof-of-Stake (PoS)
+description: Eine Erklärung des Proof-of-Stake-Konsensprotokolls und seiner Rolle in Ethereum.
+lang: de
+sidebar: true
+incomplete: true
+---
+
+Ethereum bewegt sich vom [Proof-of-Work (PoW)](/developers/docs/consensus-mechanisms/pow/) zu einem Konsensmechanismus, genannt Proof-of-Stake (PoS). Das war immer der Plan, denn es ist ein wichtiger Teil der Strategie der Community, Ethereum über [Upgrades](/upgrades/) zu skalieren. Allerdings ist es eine große technische Herausforderung, PoS richtig umzusetzen, und nicht so einfach wie PoW zu nutzen, um einen Konsens über das Netzwerk zu erzielen.
+
+## Voraussetzungen {#prerequisites}
+
+Um diese Seite besser zu verstehen, empfehlen wir dir, zuerst einen Blick auf [Konsensmechanismen](/developers/docs/consensus-mechanisms/) zu werfen.
+
+## Was ist Proof-of-Stake (PoS)? {#what-is-pos}
+
+Proof-of-Stake ist eine Art von [Konsensmechanismus](/developers/docs/consensus-mechanisms/), die von Blockchain-Netzwerken benutzt wird, um einen verteilten Konsens zu erreichen.
+
+Es verlangt von Nutzern das Einsetzen ([engl.] Staking) ihrer ETH, um ein Validator im Netzwerk zu werden. Validatoren sind verantwortlich für die gleichen Dinge wie Miner in [Proof-of-Work](/developers/docs/consensus-mechanisms/pow/): Ordnen von Transaktionen und Erstellen von neuen Blöcken, so dass alle Nodes mit dem Status des Netzwerks übereinstimmen.
```

Comment:
Should we just remove these?

Reply:
I would say yes. I've removed the English words in brackets in Crowdin, so they should probably be removed here as well.